_includes/google-analytics.html

```diff
@@ -0,0 +1,10 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
@karreiro who is the owner of this GA account?

Reply:
Hi @ederign, thank you for the review.

I'm the owner. The previous version doesn't have a Google Analytics key that I could reuse, so I created one.

We can change this by editing the key [here](https://github.com/karreiro/kiegroup-website-fork/blob/new-website/_config.yml#L8) :-)

Reply:
@karreiro it's fine for now, but I think you should start a thread with Edson/Mark to see who will be the owner of this.